### PR TITLE
10 homepage testimonials block

### LINKED ISSUE #10: [Homepage] Testimonials Block
This section needs to have a light gray background

Screenshot 2024-07-12 at 16 50 05

### PR DESCRIPTION
**Pull request by mind-adobe**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #10 

Test URLs:
- Before: https://main--piramal--aemsites.hlx.page
- After: https://10-homepage-testimonials-block--piramal--aemsites.hlx.page

### CODE REVIEW

**Review by asthabh23 — CHANGES_REQUESTED** [review]
please merge main branch changes in here and update the fonts used in this block for name/title/description.
The title shouldn't be italic. please compare once with original site again.

**Comment by asthabh23** [comment]
still needs a padding fix for the section 

Screenshot 2024-07-23 at 15 50 09

**Comment by asthabh23** [comment]
@mind-adobe yes, we need to add them!
I was about to mention that

**Comment by asthabh23** [comment]
For section padding - I will create a separate issue then so it can be taken care of in one shot

**Comment by asthabh23** [comment]
@mind-adobe , please see and address the issues under accessibility: https://pagespeed.web.dev/analysis/https-10-homepage-testimonials-block--piramal--aemsites-hlx-page/926ft22tc1?utm_source=psi&utm_medium=redirect&form_factor=mobile

this gets highlighted in SEO as well

**Comment by mind-adobe** [comment]
> @mind-adobe , please see and address the issues under accessibility: https://pagespeed.web.dev/analysis/https-10-homepage-testimonials-block--piramal--aemsites-hlx-page/926ft22tc1?utm_source=psi&utm_medium=redirect&form_factor=mobile
> 
> this gets highlighted in SEO as well

Fixed it, should be good now

File: blocks/testimonies/testimonies.css [hunk 1]
```diff
@@ -0,0 +1,290 @@
...
+  height: 205px;
+  border-radius: 50%;
+  background-color: #ed8c90;
+  animation: scale-in 6s infinite cubic-bezier(0.1, 0.21, 0.89, 0.32);
+}
+
+.block.testimonies .testimony.selected .animation-circle-0 {
+  /* outline: 20px solid #ed8c90; */
+
+  /* opacity: 0.125; */
```
Comment (asthabh23) [inline blocks/testimonies/testimonies.css:117]: please remove unrequired comments

**Review by asthabh23 — APPROVED** [review]
please remove unrequired comments before merging, rest looks good!

**Pull request merged**